api/src/test/java/org/pmiops/workbench/api/ConceptSetsControllerTest.java

```diff
@@ -1450,7 +1453,7 @@ private List<Criteria> createExpectedCriteria(
   }
 
   private void assertConceptSets(ConceptSet actual, ConceptSet expected) {
-    assertThat(actual.getCriteriums()).containsAllIn(expected.getCriteriums());
+    assertThat(actual.getCriteriums()).containsAtLeastElementsIn(expected.getCriteriums());
```

Comment:
Should this one be "containsExactly..." as well?

Reply:
quite possibly.  I wanted to be cautious here since I'm unfamiliar.

The other changes seemed more obviously correct to me.